css/_vertical_filmstrip_overrides.scss

```diff
@@ -69,10 +80,11 @@
             flex: 1;
             flex-direction: column;
             height: auto;
-            overflow-x: hidden !important;
+            justify-content: flex-end;
 
             .remote-videos-container {
-                flex-direction: column;
+                flex-direction: column-reverse;
+                overflow-x: hidden !important;
```

Comment:
I don't think the !important is needed. In general !importants aren't needed for self-controlled css and they're tricky to workaround, although it isn't likely to become a problem. I still feel it's better to avoid the finality of !important. Instead BEM or increasing specificity might be warranted.